itemadapter/utils.py

```diff
@@ -1,6 +1,44 @@
+from types import MappingProxyType
 from typing import Any
 
 
+__all__ = [
+    "get_class_field_meta",
+    "is_attrs_instance",
+    "is_dataclass_instance",
+    "is_item",
+    "is_scrapy_item",
+]
+
+
+try:
+    import scrapy
+except ImportError:
+    _scrapy_item_classes = ()  # type: tuple
+else:
+    try:
+        BaseItem = getattr(scrapy.item, "_BaseItem", scrapy.item.BaseItem)  # deprecated
+        _scrapy_item_classes = (scrapy.item.Item, BaseItem)
+    except AttributeError:
+        _scrapy_item_classes = (scrapy.item.Item,)
+
+
+def _is_dataclass_subclass(obj: Any) -> bool:
+    try:
+        import dataclasses
+    except ImportError:
+        return False
+    return dataclasses.is_dataclass(obj)
+
+
+def _is_attrs_subclass(obj: Any) -> bool:
```

Comment:
Users are not subclassing from attrs or dataclasses - what do you think about naming it `_is_attrs_class`, `_is_dataclass`?